Resources/Documentation/dv_info.md

```diff
@@ -1,32 +1,126 @@
+
+4. Click the "Repack" to repack your tape (fast forward the tape all the way and then rewind it). This button will not respond if "Deck Control" is disabled. The status of the deck (i.e. rewinding or fast forwarding) will be displayed in the "Status" field as the tape is repacked.
+
+5. To preview the tape, click on the "Playback" button in the bottom right of the "Settings" window. The settings menu will close and the Terminal window will display the settings you have entered.
+
+6. A mpv window will open displaying the playback of the tape. It may take ~20 seconds for the deck to respond and begin playing.
+> **_NOTE:_** If the timecode does not start at the very beginning of the tape, the playback window will not open until the timecode is detected. As soon as the counter starts moving on your deck and in the Terminal window, the viewer window should pop-up and begin playback. ![dv_vrecord_timecode-record-start](../dv_vrecord_timecode-record-start.gif)
+
+7. To end playback, click the `x` in the upper left hand corner of the playback window or hit the "esc" key.
+
+
+## Recording
+
+![dv_vrecord_record-instructions](../dv_vrecord_record-instructions.mp4)
+
+1. Select the name of the DV device you want to use from the "Select a DV Device" list.
+
+2. Click the "deck control" button to allow vrecord to control your device.
+
+3. To record, select the recording directory (location where you want to save the file) and enter the recording name, name of the person digitizing this tape, recording time in minutes (if applicable) and any other options you would like applied to your file.
+
+4. Click the "Record" button. The settings menu will close and the Terminal window will display the settings you have entered.
 
+5. When you are ready hit the "Return" button to begin the recording.
+
+6. A mpv window will open displaying the playback of the tape once dvrescue starts the recording. 
+> **_NOTE:_** If the timecode does not start at the very beginning of the tape, the record mode viewer will not open until the timecode is detected (as soon as the counter starts moving on your deck, the viewer window should pop-up). Please see the "Playback" section for an example.
+
+7. When dvrescue encounters an error, the software will automatically rewind and attempt to recapture the problematic frames.
+
+8. To begin another capture on a separate deck, open a new Terminal window and run `vrecord -e`.
+   - Select a different device from the list in the "DV" tab.
+   - Repeat steps 2-7 listed above.  
+
+9. Typically, vrecord will end the capture once the recording ends. If you entered a duration for the capture in "Settings," vrecord will end the capture when it reaches that duration. To end the capture manually, click the `x` in the upper left hand corner of the playback window or hit the "esc" key.
```

Comment:
Let's change last sentence to `To end the capture manually, close the playback window or hit the "esc" key.` (Not all systems vrecord runs on have things in the upper left)

Reply:
good to know! I fixed this for the playback section as well!